great_expectations/checkpoint/actions.py

```diff
@@ -1045,6 +1101,24 @@ class APINotificationAction(ValidationAction):
 
     url: str
 
+    @override
+    def v1_run(self, checkpoint_result: CheckpointResult) -> str:  # type: ignore[override] # signature does not match parent
+        aggregate_payload = []
+        for run_id, run_result in checkpoint_result.run_results.items():
+            suite_name = run_result.suite_name
+            serializable_results = convert_to_json_serializable(run_result.results)
+            batch_identifier = run_id.batch_identifier
+
+            payload = self.create_payload(
+                data_asset_name=batch_identifier,
+                suite_name=suite_name,
+                validation_results_serializable=serializable_results,
+            )
+            aggregate_payload.append(payload)
+
+        response = self.send_results(aggregate_payload)
+        return f"Successfully Posted results to API, status code - {response.status_code}"
```

Comment:
I would remove `Successfully` here since the status code may indicate a failure.